src/libraries/sendtohelix-mobile.targets

```diff
@@ -58,17 +58,22 @@
       <_XHarnessAppleCustomCommand Condition="'$(NeedsiOSSDK)' == 'true'">
         source build-apple-app.sh
       </_XHarnessAppleCustomCommand>
+      <ILLinkDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', '..', '.packages', 'microsoft.net.illink.tasks', '$(MicrosoftNETILLinkTasksVersion)'))</ILLinkDir>
```

Comment:
Are we always using `.packages` as the location for restored NuGets? 
Just wondering if it is maybe set somewhere to a property.

Reply:
There is the `NuGetPackageRoot` property that should be used.